README.md

```diff
@@ -60,11 +60,13 @@ __semantic-release-cli performs the following steps:__
 	* Add or overwrite the [`repository` field](https://docs.npmjs.com/files/package.json#repository)
 
 ## Travis CI
+
 For Travis CI, `semantic-release-cli` performs the following additional steps:
 1. Overwrite your `.travis.yml` file
 	* `after_success`: `npm install -g travis-deploy-once` and `travis-deploy-once "npm run semantic-release"`: run `semantic-release` exactly once after all builds pass
 	* Set other sane defaults: `cache: directories: ~/.npm`, `notifications: email: false`
-1. Login to Travis CI to configure the package
+1. Login to Travis CI to configure the package. This step requires the package.json [`repository` field](https://docs.npmjs.com/files/package.json#repository)
+to reference an existing Github repository.
```

Comment:
is the `repository` field case sensitive now or not? If it is I’d mention it here

Reply:
Yes, it is case-sensitive for the Travis APIs. I'll update the document.